runtime/service_integ_test.go

```diff
@@ -1003,3 +1003,58 @@ func TestDriveMountFails_Isolated(t *testing.T) {
 		assert.Error(t, err, "unexpectedly succeeded in creating a VM with a drive mount under banned path")
 	}
 }
+
+func TestUpdateVMMetadata_Isolated(t *testing.T) {
+	prepareIntegTest(t)
+
+	testTimeout := 600 * time.Second
+	ctx, cancel := context.WithTimeout(namespaces.WithNamespace(context.Background(), defaultNamespace), testTimeout)
+	defer cancel()
+
+	client, err := containerd.New(containerdSockPath, containerd.WithDefaultRuntime(firecrackerRuntime))
+	require.NoError(t, err, "unable to create client to containerd service at %s, is containerd running?", containerdSockPath)
+	defer client.Close()
+
+	pluginClient, err := ttrpcutil.NewClient(containerdSockPath + ".ttrpc")
+	require.NoError(t, err, "failed to create ttrpc client")
+	fcClient := fccontrol.NewFirecrackerClient(pluginClient.Client())
+
+	rootfsPath := defaultVMRootfsPath
+	_, err = fcClient.CreateVM(ctx, &proto.CreateVMRequest{
+		VMID: "1",
+		MachineCfg: &proto.FirecrackerMachineConfiguration{
+			MemSizeMib: 512,
+		},
+		RootDrive: &proto.FirecrackerRootDrive{
+			HostPath: rootfsPath,
+		},
```

Comment:
nit: no need to specify this, the runtime's default is the same value

Reply:
I tried removing this but it fails to create the vm if I do, saying it tries to stat the hostpath and it's "". I changed it to not use rootfsPath and just use the defaultVMRootFSPath directly though.